src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,21 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
 public class HelloWorld {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
 
+        Figure[] figures = new Figure[6];
```

Comment:
The creation of the 'figures' array is fine, but ensure that the length of the array aligns with the requirements of your task. If the number '6' is a significant value, consider making it a named constant.